tests/test_wave_01.py

```diff
@@ -118,12 +118,10 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # Assert
     assert len(updated_data["watchlist"]) is 0
     assert len(updated_data["watched"]) is 1
-    
-    # *******************************************************************************************
-    # ****** Add assertions here to test that the correct movie was added to "watched" **********
-    # *******************************************************************************************
+    assert updated_data["watched"][0]["title"] == MOVIE_TITLE_1
```

Comment:
Nice test case addition using `==`! As we mentioned in [this Slack message](https://adac17seattle.slack.com/archives/C02SM8GJUAK/p1647632889131419), but regarding the previous two lines (and more), it is more customary to use `==` to check for equality, unless the case we're checking if something is None and we'd want to use `is`.